test/integration/scheduler/podsready/suite_test.go

```diff
@@ -35,22 +38,30 @@ import (
 	workloadjob "sigs.k8s.io/kueue/pkg/controller/workload/job"
 	"sigs.k8s.io/kueue/pkg/queue"
 	"sigs.k8s.io/kueue/pkg/scheduler"
-	utiltesting "sigs.k8s.io/kueue/pkg/util/testing"
 	"sigs.k8s.io/kueue/test/integration/framework"
 	//+kubebuilder:scaffold:imports
 )
 
 var (
-	cfg       *rest.Config
-	k8sClient client.Client
-	ctx       context.Context
-	fwk       *framework.Framework
+	cfg              *rest.Config
+	k8sClient        client.Client
+	ctx              context.Context
+	fwk              *framework.Framework
+	podsReadyTimeout time.Duration
```

Comment:
I think this variable makes more sense to live in podsReadyTimeout, where it's actually setup.

Same for the clusterqueues below

Reply:
Indeed, moved all the variables that are only used in `scheduler_test`